packages/dev-app/src/views/GridView.tsx

```diff
@@ -8,15 +8,19 @@ const GridCellExample = ({ text, children }: any) => (
 	</div>
 );
 
+const test = false;
+
 export const GridView = () => {
 	return (
 		<>
 			<h1>Grid</h1>
 			<h2>Basics</h2>
 			<Grid>
-				<GridItem cols={12}>
-					<GridCellExample text="💯" />
-				</GridItem>
+				{test && (
```

Comment:
I'm not sure this should pass the type def - `test` is false (not null or undefined) which isn't one of the allowed prop types.